go.mod

```diff
@@ -3,23 +3,22 @@ module github.com/google/certificate-transparency-go
 go 1.16
 
 require (
-	bitbucket.org/creachadair/shell v0.0.6 // indirect
```

Comment:
This is disappearing again, expected?

Reply:
well, "expected" :) 
(the CI build works as-is)

Reply:
Hahaha! Well, as long as the CI is happy... 😃